src/api/README.md

```diff
@@ -1,6 +1,6 @@
 # Export Insurance Policies API
 
-CRUD API for EXIP applications. This API is built with [keystone](https://keystonejs.com/).
+GraphQL API for EXIP accounts and applications. This API is built with [Keystone](https://keystonejs.com/).
```

Comment:
isn't it better to keep this as API for EXIPs (incase it does more than accounts and applications etc)

Reply:
Good thinking, I think it's better to have a description though? I can't envision the API doing something unrelated to accounts or applications - if it does some other major piece of functionality that's not account or application related, we can just update the description?